shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/ShenyuClientRegisterServiceImpl.java

```diff
@@ -341,7 +341,8 @@ private void saveOrUpdateMetaData(final MetaDataDO exist, final MetaDataRegister
         }
         // publish MetaData's event
         eventPublisher.publishEvent(new DataChangedEvent(ConfigGroupEnum.META_DATA, eventType,
-                Collections.singletonList(MetaDataTransfer.INSTANCE.mapRegisterDTOToEntity(metaDataDTO))));
+                Collections.singletonList(MetaDataTransfer.INSTANCE.mapToData(
+                        MetaDataTransfer.INSTANCE.mapRegisterDTOToEntity(metaDataDTO)))));
```

Comment:
is used MetaDataTransfer.INSTANCE.mapToData(metaDataDO)),

Although there are type errors here, they do not lead to generic exception in   DataChangedEventDispatcher  like this  

```
 case META_DATA:
                    listener.onMetaDataChanged((List<MetaData>) event.getSource(), event.getEventType());
```